src/logic/cookies/utils/index.ts

```diff
@@ -4,9 +4,10 @@ import { getNetworkName } from 'src/config'
 
 const PREFIX = `v1_${getNetworkName()}`
 
-export const loadFromCookie = async (key: string): Promise<undefined | Record<string, any>> => {
+export const loadFromCookie = async (key: string, withoutPrefix = false): Promise<undefined | Record<string, any>> => {
```

Comment:
I can't see this function used in this PR. Why was this change necessary?

Reply:
`any` -> `unknown`

Reply:
yes, I know, but if I set it as unknown it starts to break in other places and I didn't want to start fixing them in this PR.